### PR TITLE
feat: remove public stats #4641

### LINKED ISSUE #4641: [FEATURE] Remove public stats
### Description

User must log in to see their stats, this avoids people gaming the system

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by eddiejaoude**
## Fixes Issue

closes #4641

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4642"></a>

### CODE REVIEW

File: stories/components/user/UserLink.stories.js [hunk 1]
```diff
@@ -10,9 +10,8 @@ export const Basic = {
       url: "https://www.youtube.com",
       clicks: 2,
       name: "My youtube channel",
-      icon: "FaYoutube"
+      icon: "FaYoutube",
```
Comment (krshkun) [inline stories/components/user/UserLink.stories.js:13]: Is this also formatting?

File: tests/setup/global-setup.js [hunk 2]
```diff
@@ -2,28 +2,26 @@ import { chromium } from "@playwright/test";
...
 
 const wcagUser = {
   name: "_TEST-WCAG-USER",
-  displayStatsPublic: true,
   type: "personal",
   bio: `Bio for _test-wcag-user`,
   avatar: "https://github.com/eddiejaoude.png",
-  links: links
-}
+  links: links,
```
Comment (krshkun) [inline tests/setup/global-setup.js:19]: Welp, formatting again

**Review by krshkun — COMMENTED** [review]
Also: Do we use the abbreviate Number function anywhere else since it's not removed?

File: stories/components/user/UserLink.stories.js (hunk 1 above)
Comment (eddiejaoude) [reply stories/components/user/UserLink.stories.js:13]: yes must be

File: tests/setup/global-setup.js (hunk 2 above)
Comment (eddiejaoude) [reply tests/setup/global-setup.js:19]: yep, but I won't let it block this PR 😜

**Review by krshkun — DISMISSED** [review]
Looks great

**Comment by eddiejaoude** [comment]
> I fixed the test automation to make it pass.

thank you @dan-mba 🔥 

> There are 2 images on the index page that show the profile with stats.

oh good spot 💯

**Pull request merged**